src/app/system/wrappers/CachedImage.tsx

```diff
@@ -1,22 +1,62 @@
+import { useRef } from "react"
 import { Image, ImageProps } from "react-native"
+import Animated, {
+  Easing,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from "react-native-reanimated"
 import { ImagePlaceholder } from "app/components/ImagePlaceholder"
 import { useLocalUri } from "app/system/sync/fileCache"
 
 interface CachedImageProps extends Omit<ImageProps, "source"> {
   placeholderHeight: number | null | undefined
   uri: string | undefined
+  fadeInOnLoad?: boolean
 }
 
 export const CachedImage: React.FC<CachedImageProps> = ({
+  fadeInOnLoad = true,
   placeholderHeight,
+  style,
   uri,
   ...restProps
 }) => {
+  const isDoneLoading = useRef(false)
+  const opacity = useSharedValue(fadeInOnLoad ? 0 : 1)
+  const animatedStyle = useAnimatedStyle(() => ({ opacity: opacity.value }), [])
```

Comment:
```suggestion
  const fadeInAnim = useAnimatedStyle(() => ({ opacity: opacity.value }), [])
```
this is a bit nicer as a convention